### PR TITLE
chore: optimize language switcher

### LINKED ISSUE #127: Change the language button
Currently, the `language` button at the foot of our documentation website only supports to jump to the home page of the other language. 
We hope that by clicking this button, user can jump to the other language version of the current page . If the current page doesn't have a corresponding page in another language, we can prompt certain messages and then direct users to the home page.
image

### PR DESCRIPTION
**Pull request by YiniXu9506**
Resolve #127.

### CODE REVIEW

File: src/styles/components/footer.scss [hunk 1]
```diff
@@ -131,6 +131,48 @@
...
+
+        .tooltiptext::after {
+          content: "";
+          position: absolute;
+          top: 100%;
+          left: 20%;
+          margin-left: -5px;
+          border-width: 5px;
+          border-style: solid;
+          border-color: #707070 transparent transparent transparent;
```
Comment (g1eny0ung) [inline src/styles/components/footer.scss:161]: `border-top-color` will more concise.

File: src/pages/tools.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ import { FormattedMessage } from 'react-intl'
 
 const Tools = ({ pageContext: { locale } }) => {
   return (
-    <Layout locale={locale}>
+    <Layout locale={locale} langSwitchable={true}>
```
Comment (g1eny0ung) [inline src/pages/tools.js:13]: Is the tools page also needed?

Comment (YiniXu9506) [reply src/pages/tools.js:13]: Yes, the tool page also need language switchable

File: src/styles/components/footer.scss (hunk 1 above)
Comment (YiniXu9506) [reply src/styles/components/footer.scss]: Done!

**Pull request merged**